spyder/plugins/completion/kite/plugin.py

```diff
@@ -46,27 +49,32 @@ def http_client_ready(self, languages):
         self.sig_plugin_ready.emit(self.COMPLETION_CLIENT_NAME)
 
     def send_request(self, language, req_type, req, req_id):
-        if language in self.available_languages:
+        if self.enabled and language in self.available_languages:
             self.client.sig_perform_request.emit(req_id, req_type, req)
+        else:
+            self.sig_response_ready.emit(self.COMPLETION_CLIENT_NAME,
+                                         req_id, {})
 
     def start_client(self, language):
         return language in self.available_languages
 
     def start(self):
+        # Always start client to support possibly undetected Kite builds
+        self.client.start()
         installed, path = check_if_kite_installed()
         if installed:
             logger.debug('Kite was found on the system: {0}'.format(path))
             running = check_if_kite_running()
             if not running:
                 logger.debug('Starting Kite service...')
                 self.kite_process = run_program(path)
-            self.client.start()
 
     def shutdown(self):
         self.client.stop()
         if self.kite_process is not None:
             self.kite_process.kill()
 
     def update_configuration(self):
-        enable_code_snippets = CONF.get('lsp-server', 'code_snippets')
-        self.client.enable_code_snippets = enable_code_snippets
+        self.client.enable_code_snippets = \
+            CONF.get('lsp-server', 'code_snippets')
```

Comment:
We prefer to avoid the usage of line continuations in new work. So please change the two lines above to be

```python
self.client.enable_code_snippets = CONF.get('lsp-server',
    'code_snippets')
```

Reply:
fixed!